src/Opaleye/Internal/PrimQuery.hs

```diff
@@ -53,6 +53,7 @@ data PrimQuery' a = Unit
                               (Bindings HPQ.PrimExpr)
                               (PrimQuery' a)
                               (PrimQuery' a)
+                  | Antijoin  (PrimQuery' a) (PrimQuery' a)
```

Comment:
Well in relational theory an antijoin is a thing:

> The antijoin, written as R ▷ S where R and S are relations, is similar to the semijoin, but the result of an antijoin is only those tuples in R for which there is no tuple in S that is equal on their common attribute names.

`WHERE NOT EXISTS` is a little bit of an abuse of the terminology, but then all joins were originally meant to be on common attributes rather than a general predicate, so that abuse isn't exactly uncommon